docs/DeveloperGuide.md

```diff
@@ -51,9 +51,19 @@ For example, the `Logic` component (see the class diagram given below) defines i
 
 **How the architecture components interact with each other**
 
-The *Sequence Diagram* below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The *Sequence Diagram*s below show how the components interact with each other for the scenario where the user issues the commands 
 
-<img src="images/ArchitectureSequenceDiagram.png" width="574" />
+* `delete 1` (in deliverable or meeting mode)
+
+![`Architecture Sequence Diagram with Dashboard](images/ArchitectureSequenceDiagramWithDb.png)
+
+* `delete 1` (in contact mode)
+
+![`Architecture Sequence Diagram without Dashboard](images/ArchitectureSequenceDiagram.png)
+
+* `switch m` (in any mode)
+
+![`Architecture Sequence Diagram for switch command](images/ArchitectureSequenceDiagramSwitch.png)
```

Comment:
Might not need cos covered in my Implementation section below
Since this is meant to be an overview, I feel that we do not need to cover every case

Reply:
I seee, okaokay i will remove it